packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -41,8 +42,17 @@ governing permissions and limitations under the License.
   border-right-width: 1px;
   border-right-style: solid;
 }
+.spectrum-Table-headCellContents {
+  display: inline-block;
+  flex: 1 1 auto;
```

Comment:
Do we need `flex: 1 1 auto` here? The sort icon is now positioned really far away from the column title: 
![image](https://user-images.githubusercontent.com/9661127/132901374-49ab6079-697f-4c2a-ba79-ffca91ec1442.png)

Reply:
@LFDanLu - Why has that column expanded so much? Does it have a large width set?

Reply:
In this particular [story](https://github.com/adobe/react-spectrum/blob/7b6f94a9771d3486240516992c58463405ccae54/packages/%40react-spectrum/table/stories/Table.stories.tsx#L1012-L1017) every other column is given a static width so the "Title" column takes the remaining horizontal space remaining. The sort icon IMO should always be positioned next to the column title as it was prior to this change.

Reply:
Got it. Yes, agreed that the sort icon should be closer to the title 💯

Reply:
we can just use `justify-self: start`
good catch